packages/devtools/src/commands/getElementProperty.js

```diff
@@ -0,0 +1,5 @@
+import getElementAttribute from './getElementAttribute'
+
+export default async function getElementProperty (...args) {
+    return getElementAttribute.apply(this, args)
```

Comment:
@christian-bromann we can't use `getElementAttribute` here.

It won't work, for example, when getting `value`.
`<input value="qwe">`
lets type _foobar_, see `value` attribute, it is still _qwe_, however input.value is _foobar_

See also https://pptr.dev/#?product=Puppeteer&version=v1.19.0&show=api-jshandlegetpropertypropertyname